works.css

```diff
@@ -19,16 +8,12 @@
   padding-left: 20px;
 }
 
-.project__photo {
-  width: 100%;
-}
-
 .project__title {
   margin: 0;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 300;
```

Comment:
[OPTIONAL] Please use correct font-weight for your project title, It will help the screen readers and improve the accessibility of the page and highlight our projects too
```
.project__title {
  font-weifht: 700;
}
```
your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200113567-80ed91b9-87ae-4f86-8690-7372c501f349.png)
figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200113581-4927b242-24de-4fe2-b42b-f727a942b11b.png)